brightin-template

```diff
@@ -185,19 +186,21 @@ module BrightinTemplate
       copy_file 'coffee-script/coffeelint.rake', 'lib/tasks/coffeelint.rake'
     end
 
+    def install_npm_package
+      if npm? && options[:install]
+        system 'npm install --save-dev coffeelint'
+      else
+        say "You should install coffeelint globally:\n\n  npm install -g coffeelint"
+      end
+    end
+
```

Comment:
@avdgaag These `coffeelint` adjustments seem to be unrelated to `eslint`, and a first glance, the changes here seem to be incomplete? Maybe I'm missing something, just wanted to let you know.